examples/Cassie/osc/high_level_command.cc

```diff
@@ -33,195 +34,228 @@ using drake::multibody::JacobianWrtVariable;
-
-    //////////// Get desired horizontal vel ////////////
-    // Calculate the current-desired yaw angle difference
-    double desired_filtered_yaw =
-        (1 - weight) * approx_pelvis_yaw + weight * desired_yaw;
-    double heading_error = desired_filtered_yaw - approx_pelvis_yaw;
-
-    // Apply walking speed control only when the robot is facing the target
-    // position.
-    if (heading_error < M_PI / 2 && heading_error > -M_PI / 2) {
-      // Extract quaternion from floating base position
-      Quaterniond Quat(q(0), q(1), q(2), q(3));
-      Quaterniond Quat_conj = Quat.conjugate();
-      Vector4d quat(q.head(4));
-      Vector4d quad_conj(Quat_conj.w(), Quat_conj.x(), Quat_conj.y(),
-                         Quat_conj.z());
-
-      // Calculate local target position and com velocity
-      Vector3d global_com_pos_to_target_pos_3d;
-      global_com_pos_to_target_pos_3d << global_com_pos_to_target_pos, 0;
-      Vector3d local_com_pos_to_target_pos = drake::math::quatRotateVec(
-          quad_conj, global_com_pos_to_target_pos_3d);
-      Vector3d local_com_vel = drake::math::quatRotateVec(quad_conj, com_vel);
-
-      // Sagital plane position PD control
-      double com_vel_sagital = local_com_vel(0);
-      double des_sagital_vel =
-          kp_pos_sagital_ *
-              (local_com_pos_to_target_pos(0) + target_pos_offset_) +
-          kd_pos_sagital_ * (-com_vel_sagital);
-      des_sagital_vel = std::min(vel_max_sagital_,
-                                 std::max(vel_min_sagital_, des_sagital_vel));
-
-      // Frontal plane position PD control.
-      double com_vel_lateral = local_com_vel(1);
-      double des_lateral_vel =
-          kp_pos_lateral_ * (local_com_pos_to_target_pos(1)) +
-          kd_pos_lateral_ * (-com_vel_lateral);
-      des_lateral_vel = std::min(vel_max_lateral_,
-                                 std::max(vel_min_lateral_, des_lateral_vel));
-
-      discrete_state->get_mutable_vector(des_horizontal_vel_idx_)
-              .get_mutable_value()
-          << des_sagital_vel,
-          des_lateral_vel;
-    } else {
-      discrete_state->get_mutable_vector(des_horizontal_vel_idx_)
-              .get_mutable_value()
-          << 0,
-          0;
-    }
-  }
-
-  return EventStatus::Succeeded();
-}
-
-void HighLevelCommand::CopyHeadingAngle(const Context<double>& context,
-                                        BasicVector<double>* output) const {
-  double desired_heading_pos =
-      context.get_discrete_state(des_yaw_vel_idx_).get_value()(0);
-
-  // Assign
-  output->get_mutable_value() << desired_heading_pos;
-}
-
-void HighLevelCommand::CopyDesiredHorizontalVel(
-    const Context<double>& context, BasicVector<double>* output) const {
-  auto delta_x_fs_3D_global =
-      context.get_discrete_state(des_horizontal_vel_idx_).get_value();
-
-  // Assign
-  output->get_mutable_value() = delta_x_fs_3D_global;
-}
-
-}  // namespace osc
-}  // namespace cassie
+    namespace cassie {
```

Comment:
nitpick, but the formatting seems off here. The codestyle we follow (see Drake codestyle: https://drake.mit.edu/code_style_guide.html) doesn't indent different namespaces. You can use the autoindent lines command in Clion to format your code automatically.